src/components/10-atoms/button/index.scss

```diff
@@ -60,17 +59,17 @@
 
 .a-button--large {
   min-height: 40px;
-  @include _text(default);
+  @include text(t3);
 
   @include breakpoint($mediaquery-xs-up) {
     min-height: 50px;
-    @include _text(medium);
+    @include text(t4);
   }
 }
 
 .a-button--small {
   min-height: 30px;
-  @include _text(small);
+  @include text(t4);
```

Comment:
text-small is `text(t4)` and text-medium is also t4 ?

Reply:
button small was 13px in the past. We don't have that anymore. So I have to use the smallest available. This is `t4`

Reply:
And the size of t4 is the same as figma says for "button small"?

Reply:
t4 is 14px and figma says 14px for button small👍🏼